site/content/contribute/mobile/push-notifications/corporate-proxy.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "Corporate Proxy Server"
+date: 2020-03-09T11:35:32
+weight: 3
+subsection: "Setup Push Notifications"
+---
+
+### How do I receive mobile push notification if my IT policy requires the use of a corporate proxy server?
```

Comment:
Depending on what we choose for the title, we may want to update this introduction to represent a more standalone page instead of an FAQ